09-Michael-A-Anderson/wk01/5-fri/MTA/js/script.js

```diff
@@ -0,0 +1,125 @@
+function stationFactory(line,name){
+    return {
+        name: name,
+        line:line
+    }
```

Comment:
you are 100% correct. It was not utilised at all, and made using indexOf() an absolute pain in the ass. I did a second comb through that utilised those objects to find an intersection instead of hard wiring it in. This feedback is in my list of things to change on the next iteration.